cardano_node_tests/utils/dbsync_queries.py

```diff
@@ -1463,3 +1471,19 @@ def query_db_size() -> int:
     with execute(query=query) as cur:
         result = cur.fetchone()
         return int(result[0])
+
+
+def query_new_constitution(txhash: str) -> tp.Generator[NewConstitutionInfoDBRow, None, None]:
+    """Query new constitution proposed in db-sync."""
+    query = (
+        "SELECT"
+        " constitution.id, constitution.script_hash, gap.type "
```

Comment:
Please add also `gap.id, gap.tx_id, gap.index` so the results can be easily filtered.